src/App.tsx

```diff
@@ -1,13 +1,22 @@
 import * as React from "react";
 import Routing from "./routing/Routing";
 
-import { ApolloProvider, ApolloClient, InMemoryCache } from "@apollo/client";
+import { ApolloProvider, ApolloClient, ApolloLink, InMemoryCache, HttpLink } from "@apollo/client";
+import { ApiEndpoint, ApiClientName } from "./components/api/ApiEndpoint";
 
 const client = new ApolloClient({
-  uri: "https://countries.trevorblades.com/graphql",
+  link: createApolloLink(),
   cache: new InMemoryCache()
 });
 
+function createApolloLink() {
+  return ApolloLink.split(
+    (operation) => operation.getContext().clientName === ApiClientName.countries,
```

Comment:
Couldn't find a proper way to cover this with tests. May try in future other ways.